hdinsight-with-edge-node/azuredeploy.json

```diff
@@ -0,0 +1,402 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "allowedValues": [ 
+                "Central US",
+                "East Asia",
+                "East US",
+                "Japan East",
+                "Japan West",
+                "North Europe",
+                "South Central US",
+                "Southeast Asia",
+                "West Europe",
+                "West US"
+                ],
+            "metadata": { 
+                "description": "The location where all azure resources will be deployed."
+            }
+        },
+        "vnetName": {
```

Comment:
Ideally this should become a variable to reduce the number of parameters but if it must stay a parameter it'd be a good idea to have a default value such as hdinsightVnet that follows the templatescenarioResourceName scheme.